components/transfer/Transfer.vue

```diff
@@ -316,7 +316,7 @@
 <script lang="ts" setup>
 import { ApiFactory } from '@kodadot1/sub-api'
 import { ALTERNATIVE_ENDPOINT_MAP, chainNames } from '@kodadot1/static'
-
+import { zipWith } from 'lodash'
```

Comment:
usually import it from specific method will give use smaller size. `'lodash/zipWith'`

Reply:
got it, fixed